src/components/atoms/Checkbox/Checkbox.styles.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components'
+
+import { CheckboxStateColor } from '@src/types/component'
+
+export const CheckboxWrapper = styled.div<CheckboxStateColor>`
+  && {
+    .jitera-checkbox-inner {
+      border-color: ${(props) => props.inactiveColor || '#1890ff'};
+    }
+    .jitera-checkbox-wrapper:hover .jitera-checkbox,
+    .jitera-checkbox:hover .jitera-checkbox-inner,
+    .jitera-checkbox-input:focus + .jitera-checkbox-inner {
+      border-color: ${(props) => props.activeColor || '#1890ff'};
+    }
+    .jitera-checkbox-checked::after {
+      border-color: ${(props) => props.activeColor || '#1890ff'};
+    }
+    .jitera-checkbox-checked .jitera-checkbox-inner {
+      border-color: ${(props) => props.activeColor || '#1890ff'};
```

Comment:
should we move `'#1890ff'` to `constant.ts`

Reply:
Yup, I'll move it